third_party/tlx/tutorials/hopper-fa-ws-pipelined-pingpong_test.py

```diff
@@ -144,6 +144,9 @@ def _attn_fwd_ws_pipelined_pingpong(sm_scale, M,  #
             if cid == 0:
                 # After issuing async_dot, Consumer 0 signals barrier 10 to unblock Consumer 1.
                 tlx.named_barrier_arrive(10, 256)
+            else:
+                # Consumer 1 signals barrier 9 to unblock Consumer 0.
+                tlx.named_barrier_arrive(9, 256)
```

Comment:
To reuse barrier 9 and 10 for qk dot, it is critical to signal arrival here which unblocks the loop iteration.